src/NServiceBus.Core/Pipeline/satellitePipelineExecutor.cs

```diff
@@ -17,7 +14,12 @@ public SatellitePipelineExecutor(IBuilder builder, SatelliteDefinition definitio
 
         public Task Invoke(MessageContext messageContext)
         {
+            messageContext.Context.Set(messageContext.TransportTransaction);
```

Comment:
No this is only needed until we have a better seam towards the persisters since they (NHibertnate) is doing a `context.GetOrCreate<TransportTransaction>`